HomeTask_9.py

```diff
@@ -1,10 +1,24 @@
+"""HomeTask_9.
+
+write script that will read "input.txt" file and find there all characters from english alphabet
+collect these characters and their positions in file
+after info collected this info as text write to "output.txt" file in such format
+####################
+<first found letter> -> pos1
+<next_letter> -> pos2
+<next_letter -> pos3
+etc
+#######################
+"""
 import re
-with open('input.txt', 'r') as inp:
+with open('input.txt', 'r') as inp:  # open the file for reading
     data = inp.read()
-    regexp = re.findall(r'[a-zA-Z]', data)
-    with open('output.txt', 'w') as out:
-        for i in regexp:
+    regexp = re.findall(r'[a-zA-Z]', data)  # find the all characters from english alphabet
+    with open('output.txt', 'w') as out:  # open the file for writing
+        for i in regexp:  # collect these characters and their positions in file
             out.write(f'{i} -> pos{regexp.index(i) + 1}\n')
```

Comment:
You see we need to find not index but real position in file   e -> 2427. Also please pay attention that letters could have duplicates and thus probably re.findall - not the best option here